pinot-common/src/main/java/com/linkedin/pinot/common/protocols/SegmentCompletionProtocol.java

```diff
@@ -51,7 +51,7 @@
    * MAX_SEGMENT_COMMIT_TIME_MS is the longest time (msecs) a server will take to complete building a segment and committing
    * it  (via a SegmentCommit message) after the server has been notified that it is the committer.
    */
-  public static final long MAX_SEGMENT_COMMIT_TIME_MS = 15000;
+  private static long MAX_SEGMENT_COMMIT_TIME_MS = 15000;
```

Comment:
This is not the default, it gets overwritten in the create() call.

I will make changes that will clarify this and put the default in one class.